MODULE.bazel

```diff
@@ -7,7 +7,7 @@ module(
 
 # Do not update to newer versions until you need a specific new feature.
 bazel_dep(name = "rules_license", version = "0.0.4")
-bazel_dep(name = "rules_python", version = "0.24.0")
+bazel_dep(name = "rules_python", version = "0.31.0")
```

Comment:
It's broken on Fedora 40 - the underlying hermetic python in use is looking for libcrypt.so.1 which it can't find.  Both the test and example fail (and I expect other things would too).  The output is like:
```
INFO: Analyzed target //tests/rpm:test_golden_debuginfo_rpm_contents (58 packages loaded, 2730 targets configured).
ERROR: /src/tests/rpm/BUILD:562:8: MakeRpm tests/rpm/test_debuginfo_rpm-1-0..rpm failed: (Exit 127): make_rpm failed: error executing MakeRpm command (from target //tests/rpm:test_debuginfo_rpm) bazel-out/aarch64-opt-exec-ST-13d3ddad9198/bin/pkg/make_rpm '--name=test_debuginfo_rpm' '--rpmbuild=/usr/bin/rpmbuild' '--debuginfo_type=fedora40' ... (remaining 8 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
/home/test/.cache/bazel/_bazel_test/f8087e59fd95af1ae29e8fcb7ff1a3dc/sandbox/processwrapper-sandbox/8/execroot/_main/bazel-out/aarch64-opt-exec-ST-13d3ddad9198/bin/pkg/make_rpm.runfiles/rules_python~~python~python_3_11_aarch64-unknown-linux-gnu/bin/python3: error while loading shared libraries: libcrypt.so.1: cannot open shared object file: No such file or directory
Target //tests/rpm:test_golden_debuginfo_rpm_contents failed to build
Use --verbose_failures to see the command lines of failed build steps.
```

Reply:
OK. So we'll do a major version and bzlmod bump when we release the next version.